### PR TITLE
Fix routing bug on reload

### PR DESCRIPTION
**Pull request by vikasprashar99**
Routing bug fixed on reload

### CODE REVIEW

File: lib/screens/loading.dart [hunk 1]
```diff
@@ -87,8 +89,9 @@ class _LoadingScreenState extends State<LoadingScreen> {
     } else if (_showLoginScreen == false) {
       StorageSharedPrefs p = new StorageSharedPrefs();
       String far = await p.getFarStatus();
-      String email = await p.getEmail();
-      if (far == "false") {
+      String email = await p.getEmail();     
+
+      if (far == null) {
```
Comment (Rahul-Vijay) [inline lib/screens/loading.dart:93]: Is this optimal ? I have a question here. What happens if `far` is `false` ?

File: lib/screens/auth/login.dart [hunk 2]
```diff
@@ -131,9 +131,11 @@ class _LoginScreenState extends State<LoginScreen> {
       // successfully verified phone number
       var bdy = json.decode(response.body);
       String token = bdy["token"];
+      String far = bdy["far"];
```
Comment (Rahul-Vijay) [inline lib/screens/auth/login.dart]: in the `bdy["far"]`, that data type returned is a `boolean`. Is it optimal to store as a string ? Or does dart cast it automatically from `boolean` to `String` type ?

**Review by Rahul-Vijay — CHANGES_REQUESTED** [review]
Take a look at my comments

File: lib/screens/auth/login.dart [hunk 3]
```diff
@@ -131,11 +131,9 @@ class _LoginScreenState extends State<LoginScreen> {
       // successfully verified phone number
       var bdy = json.decode(response.body);
       String token = bdy["token"];
-      String far = bdy["far"];
-      // store the returned token
       StorageSharedPrefs p = new StorageSharedPrefs();
       await p.setToken(token);
-      await p.setFarStatus(far);
+      // await p.setFarStatus(far);
```
Comment (Rahul-Vijay) [inline lib/screens/auth/login.dart:136]: do we need this comment ?

**Review by Rahul-Vijay — CHANGES_REQUESTED** [review]
Comment ?

File: lib/screens/loading.dart (hunk 1 above)
Comment (vikasprashar99) [reply lib/screens/loading.dart:93]: I am not getting far in response that's why its not being set to local storage

File: lib/screens/auth/login.dart (hunk 3 above)
Comment (vikasprashar99) [reply lib/screens/auth/login.dart:136]: no i will remove it

**Review by Rahul-Vijay — APPROVED** [review]
All changes done

**Pull request merged**